tmt/base.py

```diff
@@ -57,6 +59,15 @@
     }
 
 
+# See https://fmf.readthedocs.io/en/latest/concept.html#identifiers for
+# formal specification.
+class FmfIdType(TypedDict):
```

Comment:
Shouldn't this be typed in fmf, actually? :thinking:

Reply:
Probably, yes, I did not look into FMF code, TBH. Should FMF get type annotations, this can be replaced by a mere `import`.